docs/visualization/path/response_time.md

```diff
@@ -87,3 +87,44 @@ plot.show()
 The horizontal axis can be changed to `system time` or `index` by changing `plot.show()` to `plot.show(xaxis_type='index')`.
 
 `[worst - best] response time` in legends refers to the difference in response time between the worst case and the best case.
+
+## TimeSeries
+
+The following scripts generate timeseries graphs of response time. The horizontal axis means time, labeled as Time [s]. The vertical axis means `Response time` in the best case, worst case, worst_in_input case, or all case.
```

Comment:
```suggestion
The following scripts generate timeseries graphs of response time. The horizontal axis means time, labeled as Time [s]. The vertical axis means `Response time` in the best case, worst case, worst-in-input case, or all case.
```

Reply:
Fixed in https://github.com/tier4/CARET_doc/pull/215/commits/0759fa70312403a38e6333b5c059df59af3d1124.